pyro/infer/autoguide/guides.py

```diff
@@ -588,19 +588,21 @@ def get_posterior(self, *args, **kwargs):
         Returns a LowRankMultivariateNormal posterior distribution.
         """
         loc = pyro.param("{}_loc".format(self.prefix), self._init_loc)
+        scale = pyro.param("{}_scale".format(self.prefix),
+                           lambda: loc.new_full((self.latent_dim,), self._init_scale),
```

Comment:
do you want to make the init values `0.5 ** 0.5 * self._init_scale` so that initial `self._loc_scale()[1] == self._init_scale`?

Reply:
done.